near-contract-standards/src/non_fungible_token/event.rs

```diff
@@ -0,0 +1,174 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "standard")]
+#[serde(rename_all = "snake_case")]
+pub enum NearEvent {
+    Nep171(Nep171Event),
+}
```

Comment:
You don't need `NearEvent` type in the non-fungible token module. Go straight with `Nep171Event`

Reply:
My thoughts here was that we would collect all events in this enum.

Reply:
I know you copy-pasted it from indexer, but we discussed that going forward we want to move this structure up the hierarchy since it does not make sense to live under NFT module since we expect to have support for other standards as well. However, I am not sure it makes sense to have a single enum with all sorts of NEP events under a single umbrella in near-contract-standards.

Reply:
I moved the file to the top of the standards crate.  The benefit to this method is that it is taking advantage of the type name when generating the JSON, e.g. ` {"standard: "nep171"}`, and will be able to handle more as the are created.  I do think it makes sense to have a since umbrella for the standards.  

Eventually I see the standards as moving toward WebAssembly Interface types, which will need to be defined in one place.